This doesn't work as-is, because there is no recipe for the verilator dependency:

```
[INFO] Prefix Python version is: 3.12.3
[INFO] PyBOMBS Version 2.3.5
[INFO] Install python-apt to speed up apt processing.
[INFO] Phase 1: Creating install tree and installing binary packages:
[ERROR] Error fetching recipe `verilator':
Package verilator has no recipe file!
```

--- a/verilator.lwr
+++ b/verilator.lwr
@@ -27,4 +27,4 @@ satisfy:
   pacman: verilator
   port: verilator
   portage: sci-electronics/verilator
-source: wget+https://github.com/verilator/verilator/archive/refs/tags/v4.038.zip
+source: wget+https://github.com/verilator/verilator/archive/refs/tags/v4.038.tar.gz

--- a/verilator.lwr
+++ b/verilator.lwr
@@ -17,7 +17,7 @@
 # Boston, MA 02110-1301, USA.
 #
 
-category: common
+category: baseline
 description: Verilator Tool
 inherit: cmake
 satisfy:

--- a/verilator.lwr
+++ b/verilator.lwr
@@ -1,0 +1,30 @@
+#
+# This file is part of PyBOMBS
+#
+# PyBOMBS is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# PyBOMBS is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PyBOMBS; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+
+category: common
+description: Verilator Tool
+inherit: cmake
+satisfy:
+  deb: verilator
+  rpm: verilator
+  cmd: verilator
+  pacman: verilator
+  port: verilator
+  portage: sci-electronics/verilator
+source: wget+https://github.com/verilator/verilator/archive/refs/tags/v4.038.zip

